rewrite-maven/src/test/java/org/openrewrite/maven/UpgradeDependencyVersionTest.java

```diff
@@ -38,6 +38,82 @@
 
 class UpgradeDependencyVersionTest implements RewriteTest {
 
+    @Test
+    @Issue("https://github.com/openrewrite/rewrite-spring/issues/474")
+    void upgradePropertyAdditionalCheck() {
+        rewriteRun(
+          spec -> spec.recipe(new UpgradeDependencyVersion("org.springframework", "*", "5.2.x", "",
+            false, null)).expectedCyclesThatMakeChanges(2),
```

Comment:
Can we check once more whether we can reduce the amount of cycles to 1? 
Enforcing that all recipes complete in 1 cycle is something that's being looked at